### PR TITLE
James Koh HW01

### PR DESCRIPTION
**Pull request by jameskoh98**
---
name: PR template for CFSS
about: Used for students submitting a pull request for a homework assignment
title: HW01
labels: ''
assignees: ''

---

Before submitting your pull request, have you:

* Included all relevant files in your fork (e.g. R Markdown files, rendered Markdown files, images/graphs)?
* Added or updated a relevant `README.md` file describing the purpose of your repository, required packages, location of relevant files, etc.?

Be sure to include your name somewhere in the title or description of your pull request, as well as reflect on what was hard/easy, problems you solved, helpful tutorials you read, etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,22 @@
-# hw01
\ No newline at end of file
+Biography
```
Comment (bensoltoff) [inline README.md:1]: Uses the alternative syntax for constructing headers. Nice!

File: README.md [hunk 2]
```diff
@@ -1 +1,22 @@
...
+    -   Pianist: Sviatoslav Richter, Wilhelm Kempff, Glenn Gould
+    -   Violinist: Jascha Heifetz, David Oistrakh, Nathan Milstein,
+        Arthur Grumiaux
+
+![Portrait of J.S.Bach](https://upload.wikimedia.org/wikipedia/commons/thumb/6/6a/Johann_Sebastian_Bach.jpg/220px-Johann_Sebastian_Bach.jpg)
+
+Progress report
+===============
+
+Opened README.md with R Studio, made appropriate changes, specified md
```
Comment (bensoltoff) [inline README.md:20]: Nice description! Next time, include this text in your pull request as a comment, rather than in the repo itself

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |       x       |            |
| **Coding strategy**         |           |        x      |            |
| **Presentation**            |       x    |              |            |
| **Achievement, creativity** |   x        |              |            |
| **Ease of access**          |           |              |     x       |

* Avoid using the GitHub upload files feature. It is not practical once you have dozens of files to commit and track, which will be the case with the next assignment. You need to ensure you can stage/commit/push/pull from within RStudio

**Comment by kadenhan** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |    x       |              |            |
| **Coding strategy**         |   x        |              |            |
| **Presentation**            |       x    |              |            |
| **Achievement, creativity** |    x       |              |            |
| **Ease of access**          |           |       x       |            |

Remarks:

* Nice work, the code is well spaced out and easy to read

**Comment by royxiao99** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |           |       x       |            |
| **Achievement, creativity** |      x     |              |            |
| **Ease of access**          |           |       x       |            |

Remarks:

* Nice work, the code works and is kept simple and clean.
* However, more tools and styles could have been used in producing the text to make the writeup more visually interesting!

File: README.md [hunk 3]
```diff
@@ -1 +1,22 @@
...
\ No newline at end of file
+Biography
+=========
+
+Born and raised in South Korea, high school in America, rising third
+year undergrad majoring in math and economics. Classical musician
+(trying to be an adequate one). Likes reading and running.
+
+-   Favorite things
+    -   Composer: **J.S. Bach**
```
Comment (alexlahood) [inline README.md:9]: Included bold text, one of the required elements of this assignment.

File: README.md [hunk 4]
```diff
@@ -1 +1,22 @@
...
+    -   Conductor: Wilhelm Furtwangler
+    -   Pianist: Sviatoslav Richter, Wilhelm Kempff, Glenn Gould
+    -   Violinist: Jascha Heifetz, David Oistrakh, Nathan Milstein,
+        Arthur Grumiaux
+
+![Portrait of J.S.Bach](https://upload.wikimedia.org/wikipedia/commons/thumb/6/6a/Johann_Sebastian_Bach.jpg/220px-Johann_Sebastian_Bach.jpg)
```
Comment (alexlahood) [inline README.md:15]: Nice picture! another element of this assignment completed.

File: README.md (hunk 1 above)
Comment (alexlahood) [inline README.md:1]: Good title. Appropriate use of a heading.

File: README.md [hunk 5]
```diff
@@ -1 +1,22 @@
-# hw01
\ No newline at end of file
+Biography
+=========
+
+Born and raised in South Korea, high school in America, rising third
+year undergrad majoring in math and economics. Classical musician
+(trying to be an adequate one). Likes reading and running.
+
+-   Favorite things
```
Comment (alexlahood) [inline README.md:8]: Good list.

**Review by alexlahood — COMMENTED** [review]
Good work! You included 4/4 required elements for this homework!

**Pull request closed without merging**